admin_guide/managing_networking.adoc

```diff
@@ -86,19 +86,40 @@ Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
 [[admin-guide-disabling-hostname-collision]]
-== Disabling Host Name Collision Prevention For Ingress Objects
+== Disabling Host Name Collision Prevention For Routes and Ingress Objects
 
 In {product-title}, host name collision prevention for routes and ingress
-objects is enabled by default. This means that the host name in a route or
-ingress object can only be set on creation and not edited afterwards. Disabling
-host name collision prevention lets you edit a host name for ingress objects after creation.
-However, because {product-title} uses the object creation timestamp to determine
-the oldest route or ingress object for a given host name, the route or ingress
-object can hijack a host name with a newer route. This can happen if an older
-route changes its host name, or if an ingress object is introduced.
-
-This is relevant to {product-title} installations that depend upon Kubernetes
-behavior, including allowing the host names in ingress objects be edited.
+objects is enabled by default. This means that regular users can set the host
```

Comment:
s/regular users/application developers?

Is that what you mean by users?

Reply:
I struggled a bit with the terminology.  I mean OpenShift users who do not have the cluster-admin role, for which "regular users" seems like a reasonable term.  In contrast, the term "application developers" is less obviously a counterpart to "cluster administrator" and is not evocative of authorization and security roles.  However, I would like to use the preferred term, whatever it may be.

For what it's worth, `git grep` on the master branch finds 9 occurrences of "regular users" and 7 occurrences of "application developers".  It also found 5 occurrences of "non-administrator users", but that's ambiguous (non-cluster administrator, non-project administrator, or other?).

So, "regular users", "application developers", or is there another, more suitable term?

Reply:
@Miciah Hmm I see. Would it be better to just say "users without the admin role"? "Regular users" does sound a little vague. Not sure if there's the perfect title for the role you're describing. Let me know what you think.

There's another instance on line 120 as well.

Reply:
I changed both occurrences of "regular users" to "users without the *cluster-admin* role".  Does it read all right?

Would it be worthwhile to establish conventions for how to refer to users with or without a particular role, and document said conventions in `contributing_to_docs/doc_guidelines.adoc`?

Reply:
@Miciah LGTM. As for the guidlines, I'll send an email out. It may be that there's an edge case here where you're wanting to refer to users that aren't admins (because otherwise they'd have the admin role), but who also aren't developers, so I'll see what the other writers think.

Reply:
Well, they may or may not be developers.  RBAC and configurable security policies make devising clear and concise terminology a little tricky.  An organization may choose to allow all application developers to edit route hosts, or allow project administrators to do so, or define some new kind of users that have fewer (or different) privileges than the cluster administrator has but more (or different) privileges than application developers have.